data/devices/logitech-g-pro-x-superlight.device

```diff
@@ -0,0 +1,7 @@
+[Device]
+Name=Logitech G Pro X Superlight
+DeviceMatch=usb:046d:c547;usb:046d:c094
```

Comment:
`c547` is the ID of the receiver, the kernel driver export a hidraw node just for the device itself, that what we want. Can you use that instead? You can find it by either looking at the logs or using https://github.com/FFY00/lshid.

Reply:
Actually nevermind, this receiver is not yet supported in the kernel driver. This PR is stalled.

Reply:
Ah, interesting -- I have no experience with hardware, but there must be some kind of support for this receiver since this configuration file allows me to detect and configure the mouse using ratbagctl or my fork of piper: https://github.com/libratbag/piper/pull/616.

Anyhow, I'll close these PRs out for now since I don't have any more time to dedicate to them, but please feel free to use the code and SVGs for piper in the future if it's helpful!

Reply:
No worries, the problem is that different device might use the same receiver, so same ID. While it is fine for you to have use the receiver ID locally, as you do not have any devices that collide, we can't do that in the upstream.